pkg/common/loader.go

```diff
@@ -230,18 +239,22 @@ func (f FileLoader) Load() (*kubekeyapiv1alpha2.Cluster, error) {
 	}
 
 	if f.KubeSphereEnable {
-		if err := defaultKSConfig(&clusterCfg.Spec.KubeSphere, f.KubeSphereVersion); err != nil {
+		ver := normalizedBuildVersion(f.KubeSphereVersion)
+		if ver == "" {
+			return nil, errors.New(fmt.Sprintf("Unsupported Kubesphere Version: %v\n", f.KubeSphereVersion))
+		}
+		if err := defaultKSConfig(&clusterCfg.Spec.KubeSphere, ver); err != nil {
 			return nil, err
 		}
 	}
 
-	if f.KubernetesVersion != "" {
-		s := strings.Split(f.KubernetesVersion, "-")
+	if ver := normalizedBuildVersion(f.KubernetesVersion); ver != "" {
```

Comment:
`f.KubernetesVersion` is generated by cmd's flag:
https://github.com/kubesphere/kubekey/blob/7c761db76c5aed62289fda073e43c3c8283e9879/cmd/ctl/create/cluster.go#L133

So, I guess normalizing this variable may not work for the Kubernetes version specified in the config file.

What do you think about normalizing `clusterCfg.Spec.Kubernetes.Version` directly?